packages/next/src/server/getSitemapProps.tsx

```diff
@@ -0,0 +1,62 @@
+import { GetServerSidePropsContext } from 'next';
+import { createPagesSitemap, createRootSitemapIndex, handleSitemapPath } from '../middleware/sitemaps/createSitemaps.js';
+import { HandleSitemapRequestsConfig } from '../middleware/sitemaps/handleSitemapRequests.js';
+
+export async function getSitemapProps(
+  ctx: GetServerSidePropsContext,
+  config: any,
+) {
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
we might be very careful here with `ts-ignores` is there a particular rule that we don't follow?